### PR TITLE
SQL-1392: Modify script to add new versions instead of overriding the lastest version

### PR DESCRIPTION
**Pull request by nbagnard**

### CODE REVIEW

**Review by terakilobyte — APPROVED** [review]
LGTM! I'm not that familiar with `jq` so I ran the commands locally. Very impressed with the tool.

**Review by jchemburkar — APPROVED** [review]
LGTM!

File: resources/download-center/mongo-odbc-downloads.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
```
[the remark this answers is not in the document]
Comment (nbagnard) [reply resources/download-center/mongo-odbc-downloads.json:1]: oh, wow, no this is was my local file that I staged to test the task on Evergreen. I need to remove it!
Alexi is also going to think about how many releases we want to leave available and we'll probably modify the task slightly in the future.

[the remark this answers is not in the document]
Comment (terakilobyte) [reply resources/download-center/mongo-odbc-downloads.json:1]: Nice catch! I for some reason thought it was okay even though I knew we were going to start with a 1.0 release 🥇

**Comment by nbagnard** [comment]
@terakilobyte @jchemburkar

Here is what it does:
From
```
{
  "manual_link": "https://www.mongodb.com/docs/atlas/data-federation/query/query-with-sql/",
  "release_notes_link": "https://www.mongodb.com/docs/atlas/data-federation/query/sql/release-notes/",
  "versions": [
    {
      "version": "0.1.2",
      "platform": [
        {
          "name": "Windows",
          "download_link": "https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/windows/0.1.2/pkg/mongoodbc.msi"
        },
        {
          "name": "Linux (x86_64)",
          "download_link": "https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/ubuntu2204/0.1.2/release/mongoodbc.tar.gz"
        },
        {
          "name": "Linux (arm64)",
          "download_link": "https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/ubuntu2204-arm64/0.1.2/release/mongoodbc.tar.gz"
        },
        {
          "name": "MacOS (x86_64)",
          "download_link": "https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/macos/0.1.2/release/mongoodbc.dmg"
        }
      ]
    },
    {
      "version": "0.1.1",
      "platform": [
        {
          "name": "Windows",
          "download_link": "https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/windows/0.1.1/pkg/mongoodbc.msi"
        },
        {
          "name": "Linux (x86_64)",
          "download_link": "https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/ubuntu2204/0.1.1/release/mongoodbc.tar.gz"
        }
      ]
    }
  ]
}
```
with new release object being (sorry Evergreen logs here)

```
[2023/04/24 23:47:14.751]     {
[2023/04/24 23:47:14.751]       "version": "snapshot",
[2023/04/24 23:47:14.751]       "platform": [
[2023/04/24 23:47:14.751]         {
[2023/04/24 23:47:14.751]           "name": "Windows",
[2023/04/24 23:47:14.751]           "download_link": "https://translators-connectors-releases.s3.amazonaws.com/mongosql-odbc-driver/windows/snapshot/release/mongoodbc.msi"
[2023/04/24 23:47:14.751]         },
[2023/04/24 23:47:14.751]         {
[2023/04/24 23:47:14.751]           "name": "Ubuntu 22.04 (x86_64)",
[2023/04/24 23:47:14.751]           "download_link": "https://translators-connectors-releases.s3.amazonaws.com/mongosql-odbc-driver/ubuntu2204/snapshot/release/mongoodbc.tar.gz"
[2023/04/24 23:47:14.751]         }
[2023/04/24 23:47:14.751]       ]
[2023/04/24 23:47:14.751]     }

```
you get (Evergreen logs here too)
```
[2023/04/24 23:47:15.041] {
[2023/04/24 23:47:15.041]   "manual_link": "https://www.mongodb.com/docs/atlas/data-federation/query/query-with-sql/",
[2023/04/24 23:47:15.041]   "release_notes_link": "https://www.mongodb.com/docs/atlas/data-federation/query/sql/release-notes/",
[2023/04/24 23:47:15.041]   "versions": [
[2023/04/24 23:47:15.041]     {
[2023/04/24 23:47:15.041]       "version": "snapshot",
[2023/04/24 23:47:15.041]       "platform": [
[2023/04/24 23:47:15.041]         {
[2023/04/24 23:47:15.041]           "name": "Windows",
[2023/04/24 23:47:15.041]           "download_link": "https://translators-connectors-releases.s3.amazonaws.com/mongosql-odbc-driver/windows/snapshot/release/mongoodbc.msi"
[2023/04/24 23:47:15.041]         },
[2023/04/24 23:47:15.041]         {
[2023/04/24 23:47:15.041]           "name": "Ubuntu 22.04 (x86_64)",
[2023/04/24 23:47:15.041]           "download_link": "https://translators-connectors-releases.s3.amazonaws.com/mongosql-odbc-driver/ubuntu2204/snapshot/release/mongoodbc.tar.gz"
[2023/04/24 23:47:15.041]         }
[2023/04/24 23:47:15.041]       ]
[2023/04/24 23:47:15.041]     },
[2023/04/24 23:47:15.041]     {
[2023/04/24 23:47:15.041]       "version": "0.1.2",
[2023/04/24 23:47:15.041]       "platform": [
[2023/04/24 23:47:15.041]         {
[2023/04/24 23:47:15.041]           "name": "Windows",
[2023/04/24 23:47:15.041]           "download_link": "https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/windows/0.1.2/pkg/mongoodbc.msi"
[2023/04/24 23:47:15.041]         },
[2023/04/24 23:47:15.041]         {
[2023/04/24 23:47:15.041]           "name": "Linux (x86_64)",
[2023/04/24 23:47:15.041]           "download_link": "https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/ubuntu2204/0.1.2/release/mongoodbc.tar.gz"
[2023/04/24 23:47:15.041]         },
[2023/04/24 23:47:15.041]         {
[2023/04/24 23:47:15.041]           "name": "Linux (arm64)",
[2023/04/24 23:47:15.041]           "download_link": "https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/ubuntu2204-arm64/0.1.2/release/mongoodbc.tar.gz"
[2023/04/24 23:47:15.041]         },
[2023/04/24 23:47:15.041]         {
[2023/04/24 23:47:15.041]           "name": "MacOS (x86_64)",
[2023/04/24 23:47:15.041]           "download_link": "https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/macos/0.1.2/release/mongoodbc.dmg"
[2023/04/24 23:47:15.041]         }
[2023/04/24 23:47:15.041]       ]
[2023/04/24 23:47:15.041]     },
[2023/04/24 23:47:15.041]     {
[2023/04/24 23:47:15.041]       "version": "0.1.1",
[2023/04/24 23:47:15.041]       "platform": [
[2023/04/24 23:47:15.041]         {
[2023/04/24 23:47:15.041]           "name": "Windows",
[2023/04/24 23:47:15.041]           "download_link": "https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/windows/0.1.1/pkg/mongoodbc.msi"
[2023/04/24 23:47:15.041]         },
[2023/04/24 23:47:15.041]         {
[2023/04/24 23:47:15.041]           "name": "Linux (x86_64)",
[2023/04/24 23:47:15.041]           "download_link": "https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/ubuntu2204/0.1.1/release/mongoodbc.tar.gz"
[2023/04/24 23:47:15.041]         }
[2023/04/24 23:47:15.041]       ]
[2023/04/24 23:47:15.041]     }
[2023/04/24 23:47:15.041]   ]
[2023/04/24 23:47:15.041] }

```

**Comment by nbagnard** [comment]
evergreen merge

**Pull request merged**